src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp

```diff
@@ -15,125 +15,6651 @@
+                        {
+                            garrosh->SetFacingToObject(sylvanas);
+                            garrosh->PlayOneShotAnimKitId(DATA_ANIMKIT_GARROSH_1);
+                            garrosh->AI()->Talk(TALK_0);
+                        }
+                    }
+
+                    _events.ScheduleEvent(EVENT_TALK_SEQUENCE + 3, 12s);
+                    break;
+                }
+
+                case EVENT_TALK_SEQUENCE + 3:
+                {
+                    if (Creature* garrosh = ObjectAccessor::GetCreature(*me, _garroshGUID))
+                    {
+                        garrosh->SetFacingTo(3.9444442f);
+                        garrosh->AI()->Talk(TALK_1);
+                    }
+
+                    _events.ScheduleEvent(EVENT_TALK_SEQUENCE + 4, 7s);
+                    break;
+                }
+
+                case EVENT_TALK_SEQUENCE + 4:
+                {
+                    if (Creature* sylvanas = ObjectAccessor::GetCreature(*me, _sylvanasGUID))
+                    {
+                        if (Creature* warhorse = ObjectAccessor::GetCreature(*me, _warhorseGUID))
+                            warhorse->SetFacingTo(2.4260077f);
+
+                        sylvanas->AI()->Talk(TALK_2);
+                    }
+
+                    _events.ScheduleEvent(EVENT_TALK_SEQUENCE + 5, 5s);
+                    break;
+                }
+
+                case EVENT_TALK_SEQUENCE + 5:
+                {
+                    if (Creature* sylvanas = ObjectAccessor::GetCreature(*me, _sylvanasGUID))
+                    {
+                        if (Creature* warhorse = ObjectAccessor::GetCreature(*me, _warhorseGUID))
+                            warhorse->SetFacingTo(3.7350047f);
+
+                        sylvanas->PlayOneShotAnimKitId(DATA_ANIMKIT_SYLV_1);
+                        sylvanas->AI()->Talk(TALK_3);
+                    }
+
+                    _events.ScheduleEvent(EVENT_TALK_SEQUENCE + 6, 16s);
+                    break;
+                }
+
+                case EVENT_TALK_SEQUENCE + 6:
+                {
+                    if (Creature* sylvanas = ObjectAccessor::GetCreature(*me, _sylvanasGUID))
+                        sylvanas->AI()->Talk(TALK_4);
+
+                    _events.ScheduleEvent(EVENT_TALK_SEQUENCE + 7, 4s);
+                    break;
+                }
+
+                case EVENT_TALK_SEQUENCE + 7:
+                {
+                    if (Creature* garrosh = ObjectAccessor::GetCreature(*me, _garroshGUID))
+                        garrosh->AI()->Talk(TALK_2);
+
+                    _events.ScheduleEvent(EVENT_TALK_SEQUENCE + 8, 3s);
+                    break;
+                }
+
+                case EVENT_TALK_SEQUENCE + 8:
+                {
+                    if (Creature* sylvanas = ObjectAccessor::GetCreature(*me, _sylvanasGUID))
+                        sylvanas->AI()->Talk(TALK_5);
```

Comment:
honestly... at this point, you should have given sylvanas her own script and probably via sai to handle that...
also ai check.